pageserver/Cargo.toml

```diff
@@ -35,6 +35,8 @@ humantime-serde.workspace = true
 hyper.workspace = true
 itertools.workspace = true
 nix.workspace = true
+# hack to get the number of worker threads tokio uses
+num_cpus = { version = "1.15" }
```

Comment:
It is not clear to me why number of workers mostly performing IO operations is determined by number of CPUs.

Reply:
Because we do repartitioning and create_image_layers on the async workers (8), but I'll add comments.